dev-docs/bidder-adaptor.md

```diff
@@ -12,259 +11,504 @@ nav_section: adapters
 # How to Add a New Bidder Adapter
 {:.no_toc}
 
-{: .alert.alert-warning :}
-This page contains legacy instructions on how to build a Prebid adapter before Prebid 1.0. Adapter writers should use the [Prebid 1.0 adapter guidelines.]({{site.baseurl}}/dev-docs/bidder-adapter-1.html)
-
-{: .alert.alert-danger :}
-As of October 1st, 2017, we will no longer be accepting pull requests for adapters that are not compliant with Prebid 1.0 conventions.
-
 At a high level, a bidder adapter is responsible for:
 
-1. Sending out bid requests to the ad server
-2. Registering the bids that are returned with Prebid.js
+1. Creating the bid requests for the bidder's server.
+2. Parsing and registering the bid responses.
 
 This page has instructions for writing your own bidder adapter.  The instructions here try to walk you through some of the code you'll need to write for your adapter.  When in doubt, use [the working adapters in the Github repo](https://github.com/prebid/Prebid.js/tree/master/modules) for reference.
 
-{: .alert.alert-success :}
-**Adding a Video Bidder Adapter?**  
-See [How to Add a New Video Bidder Adapter]({{site.baseurl}}/dev-docs/how-to-add-a-new-video-bidder-adaptor.html).
-
 * TOC
 {:toc}
 
+## Planning your adapter
+
+With each adapter submission, there are two files required to be in the pull request:
+
+* `modules/exampleBidAdapter.js`: the file containing the code for the adapter
+* `modules/exampleBidAdapter.md`: a markdown file containing key information about the adapter:
+   * The contact email of the adapter's maintainer.
+   * A test ad unit that will consistently return test creatives. This helps us to ensure future Prebid.js updates do not break your adapter.  Note that if your adapter supports video, outstream video, or native, you must also provide example parameters for each type.
```

Comment:
Instead of "supports video, outstream video, or native", let's say something like "supports video (instream and/ or outstream context) or native..."

Reply:
Cool, updated in ac9f046